src/commands/init.ts

```diff
@@ -17,21 +20,43 @@ export default class Init extends Command {
   ]
 
   static flags = {
-    version: Flags.string({ description: 'project version' })
+    version: Flags.string({ description: 'project version' }),
+    'from-hub': Flags.boolean({ description: 'clone a bundle from Entando hub' }),
```

Comment:
1) use property name not string here as version
2) Clones a bundle from the Entando Hub

Reply:
I tried using `fromHub` as camel-cased property name thinking that it will understand to convert to dashed case, but it doesn't work. the only way I can use the flag `from-hub` is to state the property name using string

Reply:
Ok, then convert all of them to string

Reply:
just to let you know, eslint won't allow to convert the undashed property names to string. so i'll leave them be as it is